gui/include/streamsession.h

```diff
@@ -43,9 +57,8 @@ struct StreamSessionConnectInfo
 	QByteArray morning;
 	ChiakiConnectVideoProfile video_profile;
 	unsigned int audio_buffer_size;
-	bool fullscreen;
 
-	StreamSessionConnectInfo(Settings *settings, QString host, QByteArray regist_key, QByteArray morning, bool fullscreen);
+	StreamSessionConnectInfo(Settings *settings, QString host, QByteArray regist_key, QByteArray morning);
```

Comment:
^

Reply:
These fullscreen lines must be from someone else I am pretty sure I didn't add those.
As I don't know what to do I'll just leave for now.